### DISCUSSION TITLE
Added: LDM Notes for Nov 11, 2019

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#2953] [General]
https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-11-11.md

Agenda:

1.    Confirm removal of warning calling a method that returns [MaybeNull]T
2.    Allow interpolated string constant
3.    Enhancing the Common Type Specification
4.    Type pattern
 5.   Simple name binding with target types

**Comment by DarthVella** [comment]
Forgive me, but even having read the referenced issue #2926 this code makes no sense to me. Is it a typo? Or what exactly is this doing?

```csharp
class Outer
{
    public class Inner1 : Outer {}
    public class Inner2<T> : Outer {}
}

int M1(Outer o) => o switch
{
    new Inner1(); // Binds to Outer
    new Inner2<int>();
}
```

**Comment by agocke** [comment]
Typo. Fixed.
